### PR TITLE
Remove `zwave` and `ozw` integration support

### PR DESCRIPTION
**Pull request by raman325**
## Breaking change

Removes `zwave` and `ozw` support from the integration.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:

### CODE REVIEW

File: tests/test_config_flow.py [hunk 1]
```diff
@@ -384,59 +379,3 @@ async def test_options_flow_with_zwavejs(
...
         await hass.async_block_till_done()
         assert entry.data.copy() == data
-
-
-async def test_get_entities(hass, lock_data):
-    """Test function that returns entities by domain."""
-    await setup_ozw(hass, fixture=lock_data)
-
-    # Make sure the lock loaded
-    state = hass.states.get("lock.smartcode_10_touchpad_electronic_deadbolt_locked")
```
Comment (firstof9) [inline tests/test_config_flow.py:394]: Can we change this to load the zwavejs entities and verify the lock loads instead?

```python
    # Load ZwaveJS
    node = lock_kwikset_910
    state = hass.states.get(KWIKSET_910_LOCK_ENTITY)
```

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
Tested the changes locally, and they do work.

**Review by firstof9 — APPROVED** [review]

**Comment by raman325** [comment]
@firstof9 thanks for adding the tests, clearly never got a chance to do it

**Comment by rcdailey** [comment]
I saw this in the release notes and grew concerned:

- No reference from the release notes to documentation explaining how this is a breaking change or what manual actions I need to take (unless I missed it)
- Does this mean I can't use my Schlage Smart Deadbolt anymore (since it's zwave; provided via the zwavejs2mqtt plugin)?

**Comment by raman325** [comment]
> I saw this in the release notes and grew concerned:
> 
> * No reference from the release notes to documentation explaining how this is a breaking change or what manual actions I need to take (unless I missed it)
> * Does this mean I can't use my Schlage Smart Deadbolt anymore (since it's zwave; provided via the zwavejs2mqtt plugin)?

Locks using `zwave_js` will still work. Newer version of HA don't have the `zwave` or `ozw` integrations, so we got rid of the corresponding code. It's listed as a breaking change because it was worth highlighting, but it's not really a breaking change since `keymaster` is now pinned to versions of HA that no longer have those integrations anyway

**Comment by Monkey-Shiner** [comment]
> > I saw this in the release notes and grew concerned:
> > 
> > * No reference from the release notes to documentation explaining how this is a breaking change or what manual actions I need to take (unless I missed it)
> > * Does this mean I can't use my Schlage Smart Deadbolt anymore (since it's zwave; provided via the zwavejs2mqtt plugin)?
> 
> Locks using `zwave_js` will still work. Newer version of HA don't have the `zwave` or `ozw` integrations, so we got rid of the corresponding code. It's listed as a breaking change because it was worth highlighting, but it's not really a breaking change since `keymaster` is now pinned to versions of HA that no longer have those integrations anyway

Thank you for the clarification!

**Comment by lishan89uc** [comment]
> 

HAHAHAHA YOU scared the **** out of me ROFL!!!!! I thought my beautiful fe 599 will no longer work LOL!!!!

**Pull request merged**